public/.htaccess

```diff
@@ -1,25 +1,68 @@
-DirectoryIndex app.php
+# Use the front controller as index file. It serves as a fallback solution when
```

Comment:
Should we remove this file? It is available with `symfony/apache-pack` + `symfony/recipes-contrib`: https://github.com/symfony/recipes-contrib/blob/master/symfony/apache-pack/1.0/public/.htaccess

Reply:
I agree